examples/fetch/src/lib.rs

```diff
@@ -1,8 +1,3 @@
-extern crate futures;
-extern crate js_sys;
-extern crate wasm_bindgen;
-extern crate wasm_bindgen_futures;
-extern crate web_sys;
 #[macro_use]
 extern crate serde_derive;
```

Comment:
If you're extra ambitious, we can actually remove `serde_derive` both here and in `Cargo.toml` by adding this to `Cargo.toml`:

```toml
[dependencies]
serde = { version = "1.0.80", features = ["derive"] }
```

and then adding here:

```rust
use serde::{Deserialize, Serialize};
```

Reply:
@alexcrichton 
Thank you for your comment.
I'll try it !!